<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register_Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_register">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fullname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fullname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fullname">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2_username">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2_username">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3_tel_no">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3_tel_no">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>tel_no</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4_identity">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>identity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4_identity">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5_email">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5_email">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6_password">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6_password">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
